### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1082] [Galaxy Trivia] [answered]
> ☁️☁️ are not ..., 🗻🗻 are not ..., ... are not 🔵🔵, and ...

**Bonus:** Name the author.

**Comment by peterbaccaro** [comment]
**Clouds are not spheres, mountains are not cones, coastlines are not circles, and bark is not smooth, nor does lightning travel in a straight line** is a famous quotes by **Benoît Mandelbrot**.

The quote emphasizes the intricate and irregular nature of natural phenomena, which is characteristic of fractal geometry, a field that Mandelbrot pioneered. The quote highlights the diversity and complexity of the world around us, reminding us that nature's beauty often lies in its irregular and self-similar patterns.

**Comment by NomanGhiasy** [comment]
Clouds are not spheres, mountains are not cones, coastlines are not circles, and bark is not smooth, nor does lightning travel in a straight line is a famous quotes by Benoît Mandelbrot.

**Comment by thuongtruong1009** [comment, marked as answer]
Complete the expression: "Clouds ☁️☁️ are not measured, mountains 🗻🗻 are not weighed, long rains are not ... and strong winds are not ..."

Name the author: "The Book of Ecclesiastes" in the Bible. The quote is from Ecclesiastes 1:7.
